app/api/api_v1/schemas/document.py

```diff
@@ -52,21 +52,22 @@ class FamilyDocumentResponse(BaseModel):
     """Response for a FamilyDocument, without any family information"""
 
     import_id: str
-    variant: Optional[str]
+    variant: Optional[str] = None
     slug: str
     # What follows is off PhysicalDocument
     title: str
-    md5_sum: Optional[str]
-    cdn_object: Optional[str]
-    source_url: Optional[str]
-    content_type: Optional[str]
+    md5_sum: Optional[str] = None
+    cdn_object: Optional[str] = None
+    source_url: Optional[str] = None
+    content_type: Optional[str] = None
     # TODO: Remove after transition to multiple languages
     language: str
     languages: Sequence[str]
-    document_type: Optional[str]
-    document_role: Optional[str]
+    document_type: Optional[str] = None
```

Comment:
Please note: https://linear.app/climate-policy-radar/issue/PDCT-248/schema-change-for-document-type ... this was raised after a conversation with @alaniswright ... This should not be optional in the future

Reply:
Good spot, probably a separate change though?